sktime/forecasting/tests/test_all_forecasters.py

```diff
@@ -642,15 +527,23 @@ def test_pred_int_tag(self, estimator_instance):
         contains_interval_adapter = hasattr(f, "_extract_conf_int") and callable(
             f._extract_conf_int
         )
+
         implements_interval_adapter = f._has_implementation_of("_extract_conf_int")
 
         if contains_interval_adapter and not implements_interval_adapter:
             return None
 
         # check which methods are implemented
-        implements_interval = f._has_implementation_of("_predict_interval")
-        implements_quantiles = f._has_implementation_of("_predict_quantiles")
-        implements_proba = f._has_implementation_of("_predict_proba")
+        from sktime.forecasting.base import _BaseGlobalForecaster
```

Comment:
do not change this test, this is bad style, you are introducing significant coupling! Will explain how to do it better

Reply:
done